### PR TITLE
ISSUE-6 JIB with distributed james server

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: openpaas-james/apps/distributed/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# openpaas-james Distributed server
+
+This server is the distributed version of James relying on:
+
+* Cassandra
```
Comment (chibenwa) [inline openpaas-james/apps/distributed/README.md:5]: Versions please

File: openpaas-james/apps/distributed/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+docker run -d --name=elasticsearch --env 'discovery.type=single-node' docker.elastic.co/elasticsearch/elasticsearch:6.3.2
+
+docker run -d --name=tika apache/tika:1.24 #Optional
+```
+
+Then you can finally start the James distributed server:
+
+```
+docker run -d --hostname HOSTNAME -p "25:25" -p 80:80 -p "110:110" -p "143:143" -p "465:465" -p "587:587" -p "993:993" -p "8000:8000" --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link s3:s3.docker.test --name james -t linagora/openpaas-james-distributed
```
Comment (chibenwa) [inline openpaas-james/apps/distributed/README.md:41]: Use networks, not links please

Comment (Arsnael) [reply openpaas-james/apps/distributed/README.md:41]: I thought that was for just docker-compose, my bad :)

File: openpaas-james/apps/distributed/README.md (hunk 1 above)
Comment (Arsnael) [reply openpaas-james/apps/distributed/README.md:5]: Ok.... Even if I'm pretty sure those might likely get forgotten to be updated the day we upgrade those

**Review by chibenwa — APPROVED** [review]

**Review by LanKhuat — APPROVED** [review]

**Comment by chibenwa** [comment]
Squashed

**Pull request merged**